src/commands/apps/create.ts

```diff
@@ -46,6 +47,12 @@ export default class AppCreateCommand extends Command {
   @hasArg
   platform: string;
 
+  @help("The environment of the app. This must be one word, alphanumeric, first letter capitalized.")
```

Comment:
The spec says the acceptable values for this field are: Alpha, Beta, Production, Store, Enterprise.  Mentioning those here will help the user.
  
Instead of "The environment of the app", I think "The app release type" is better.

Reply:
> The spec says the acceptable values for this field are: Alpha, Beta, Production, Store, Enterprise. Mentioning those here will help the user.
Unfortunately, there are more than just these set of acceptable values. Those are the _suggested_ values - but a user can enter _any_ environment that meets the ruleset.

I'm not opposed to mentioning the values for this field - but they will make the error message significantly wordier. Is that an issue?

> Instead of "The environment of the app", I think "The app release type" is better.
I think this is problematic. HockeyApp has an existing field called `release_type` and we want to differentiate an App Center `environment` from that.

Perhaps 'The app environment...' is a better sentence starter?

Pre-emptively, I'd like to suggest something like:

'The app environment. Suggested values are Alpha, Beta, Production, Store, Enterprise. Custom values are allowed, and must be must be one word, alphanumeric, first letter capitalized.'

Thoughts?

cc: @blparr

Reply:
The spec says that no custom values are allowed.